services/apps/alcs/src/providers/typeorm/migrations/1702658459584-remove-crownlandtype-from-submission.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class RemoveCrownlandtypeFromSubmission1702658459584 implements MigrationInterface {
+    name = 'RemoveCrownlandtypeFromSubmission1702658459584'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "alcs"."application_submission" DROP COLUMN "crown_land_owner_type"`);
+        await queryRunner.query(`ALTER TABLE "alcs"."notice_of_intent_submission" DROP COLUMN "crown_land_owner_type"`);
```

Comment:
If we are reversing previous migration, do we also need to drop the column in `application_owner` and `notice_of_intent_owner`?

Reply:
no, we need those in owner tabler, not needed in submission ones